scripts/torque_controller.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/python3
+
+# Copyright 2022 Open Source Robotics Foundation, Inc. and Monterey Bay Aquarium Research Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import rclpy
+import numpy as np
+from scipy import interpolate
```

Comment:
I think we need and `<exec_depend>python3-scipy</exec_depend>` on `packages.xml`

Reply:
added exec_depend

re: breaking into separate packages, I agree. It will be much cleaner and follows convention of other packages in ROS. Thoughts @hamilton8415 ?

Reply:
> added exec_depend

I don't see it committed 🧐 

> re: breaking into separate packages,

I don't think we need to block this PR on that, it can be addressed later if needed

Reply:
odd... I see it in the andermi/python_example branch on github (62fc107), but the PR didn't pick it up and only shows 1 commit

Reply:
I changed the base away and then back to main and now it found the commit. :man_shrugging: ... "have you tried turning it off and on again?"